.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,26 @@
 #### Why we need this PR?
-*Pick one*
+*Pick one, remove the rest*
 - It fixes a bug *bug description*
 - It adds new feature needed because *why we need this feature*
 - Refactoring to improve *what*
 
 #### What happened in this PR?
- - Explain solution of the problem, new feature added.
- - What was changed, added, removed?
- - What is most important part that reviewers should focus on?
- - Was this PR tested? How?
- - Were docs and examples updated, if necessary?
+*Fill relevant points, put NA otherwise*
+ - Problem solution in this PR:
+ 
+   *[ Explain solution of the problem, new feature added here. ]*
```

Comment:
Strange formatting - shouldn't this line stick to the previous?

Reply:
Fixed